### DISCUSSION TITLE
Permanently blocked one of my sets

### DISCUSSION THREAD

**Discussion by AlphaeusNg** [proginosko/LeechBlockNG#410] [Q&A] [answered]
Hi, I accidentally perma blocked one of my sets, and now it doesn't unlock.

These were the settings I used:

```
Enter the time periods within which to block these sites (optional): 0000-2400
Enter a time limit after which to block these sites (optional): 0 minutes in every
Select when to block these sites: within time periods OR after time limit
Select the days on which to block these sites: All the days selected (Sun   Mon   Tue   Wed   Thu   Fri   Sat)
```

These are the advanced options settings
![image](https://github.com/proginosko/LeechBlockNG/assets/78588510/f9844cf4-37b6-4769-9166-f968b8769dd4)

I'm wondering if there's any way to unblock it? Thank you! #

**Comment by proginosko** [comment, marked as answer]
https://www.proginosko.com/leechblock/faq/options-disabled/

Email leechblockng@proginosko.com for a solution.
